src/psyclone/domain/lfric/lfric_arg_descriptor.py

```diff
@@ -108,9 +108,9 @@ class LFRicArgDescriptor(Descriptor):
     # indicates either x1d or y1d.
     # Note, the LFRic infrastructure currently does not have 'region' as
```

Comment:
Presumably this comment needs updating/removing?

Reply:
Removed, see commit 6b2e780d510f5c50ed82875502eb3d0a74c7099a